packages/terra-date-input/src/DateInput.jsx

```diff
@@ -442,6 +470,35 @@ class DateInput extends React.Component {
     }
   }
 
+  /**
+   * Sets state to new value dependent on existing date
+   * @param {Object} event Event object generated from the event delegation.
+   * @param {Number} addDays Adds days to current date or today's date.
+   */
+  setHotKeyDate(event, addDays) {
+    if (!this.props.value || event.keyCode === KeyCode.KEY_T) {
```

Comment:
We should use the current state of the date instead of using this.props.value. For example, if this.props.value was 01/01/2021 but the user changed it to 02/02/2020 and pressed `T`, `-`, or `+`, we would want to calculate the new date using 02/02/202 instead of the value passed in.

Reply:
Fixed here: https://github.com/cerner/terra-framework/pull/1483/commits/520855d1e17fd222b3585c138de09be411922f8a#diff-ea35ec190c52db171ef5f774b071e80db0b038abff0e19580b310959fc582329R481